pkg/env/costmodelenv.go

```diff
@@ -509,3 +511,7 @@ func GetPromClusterLabel() string {
 func IsIngestingPodUID() bool {
 	return GetBool(IngestPodUIDEnvVar, false)
 }
+
+func GetAllocationNodeLabelWhiteList() []string {
+	return GetList(AllocationNodeLabelWhiteList, ",")
```

Comment:
Can you copy the helm defaults here? Not every user uses helm to deploy and this helps standardize defaults in the off chance users use a legacy helm chart to deploy a new KC version.

Reply:
We can do that, yeah.

I was thinking that if the list is empty, that would indicate that you wanted this feature to be completely turned off. Instead do we need an `enabled` flag? And if enabled is `true`, but the list is empty, we default?

Reply:
I think it's safe to turn this feature on for everyone...do we need enabled?

Reply:
If you want empty list to mean "use defaults" and not "don't ingest any node labels" I think you need `enabled`. It looks fine to me that way! (See https://github.com/kubecost/cost-analyzer-helm-chart/pull/1925)